test/webapi/ows/stac/demo-collection.json

```diff
@@ -7,6 +7,19 @@
       ],
       "type": "application/zarr",
       "href": "http://localhost:8080/s3/datasets/demo.zarr",
+      "xcube:store_kwargs": {
+        "data_store_id": "s3",
```

Comment:
Use separate asset `xcube:data_store_id` to align it with the positional argument `data_store_id` of the data store API function `new_data_store()`.